Makefile

```diff
@@ -1,4 +1,4 @@
-.DEFAULT_GOAL := build
+.DEFAULT_GOAL := build_geth
```

Comment:
I would build everything by default. To maintain the `make && make install` philosophy.

Reply:
And so `make cmd/geth` would, as you say above, **build** geth in the default target build dir, eg `./bin/geth`?